clients/admin-ui/package.json

```diff
@@ -10,7 +10,7 @@
     "copy-export": "mkdir -p ../../src/fides/ui-build/static/admin/ && rsync -a --delete out/ ../../src/fides/ui-build/static/admin/",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:start": "next build && NODE_ENV=test next start",
+    "cy:start": "NODE_ENV=test next build && NODE_ENV=test next start",
```

Comment:
This, the `next.config` change, and the `.env.test` are all to cut down on the noise that makes debugging cypress failures harder.